clients/admin/src/account.rs

```diff
@@ -48,7 +51,21 @@ pub fn info(core: &Core, username: Option<String>, public_key: Option<String>) -
 
     let account_info = core.admin_get_account_info(identifier)?;
 
-    println!("{:#?}", account_info);
+    println!("{}", account_info.username);
+    println!("root: {}", account_info.root);
+    println!("payment_platform: {:#?}", account_info.payment_platform);
+
+    let last_active = Duration::milliseconds(
+        (SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap()
+            .as_millis() as u64
+            - account_info.last_active) as i64,
+    )
```

Comment:
+ I think we just want to `{:#?}` this
+ I don't think we want to say how long ago they were active, I think we want to print the date / time they were last active. I think if this object contained an `Instant` or something the debug print would handle that too.
+ if you need to give anything special attention, consider impl'ing `Debug` the way we do for `Trial`.